test/feedback_test.go

```diff
@@ -0,0 +1,11 @@
+package test
+
+func (s *CLITestSuite) TestFeedback() {
+	tests := []CLITest{
+		{args: "feedback", fixture: "feedback/no-confirm.golden", input: "n\n"},
+		{args: "feedback", fixture: "feedback/received.golden", input: "y\n This CLI is great!\n"},
```

Comment:
```suggestion
		{args: "feedback", fixture: "feedback/received.golden", input: "y\nThis CLI is great!\n"},
```

Reply:
I think this is for testing that preceding spaces are trimmed away.

Reply:
If so, let's unit test that instead!

Reply:
I actually don't know how to get the entered feedback to show up in the golden file. I pass it in as input to the test but it doesn't show up in the actual output of the test. Maybe it would require printing in the feedback command, which we wouldn't want to do.

Reply:
> I actually don't know how to get the entered feedback to show up in the golden file.

It currently doesn't show up... there's probably a way to do it but I haven't figured it out yet

Reply:
(you can ignore that limitation for now)